includes/class-wc-payments-onboarding-service.php

```diff
@@ -437,6 +450,37 @@ function () {
 		return $business_types;
 	}
 
+	/**
+	 * Check whether an onboarding initialization is in progress.
+	 *
+	 * This only relates to the initial account creation, not the full KYC flow.
+	 *
+	 * @return bool Whether an onboarding flow is in progress.
+	 */
+	public function is_onboarding_init_in_progress(): bool {
+		return filter_var( get_transient( self::ONBOARDING_INIT_IN_PROGRESS_TRANSIENT ), FILTER_VALIDATE_BOOLEAN );
+	}
+
+	/**
+	 * Mark the onboarding initialization as in progress.
+	 *
+	 * This only relates to the initial account creation, not the full KYC flow.
+	 *
+	 * @return void
+	 */
+	public function set_onboarding_init_in_progress(): void {
+		set_transient( self::ONBOARDING_INIT_IN_PROGRESS_TRANSIENT, 'yes', 3 * MINUTE_IN_SECONDS );
```

Comment:
Why 3 minutes? 🙂

Reply:
1 minute seemed to be cutting it close (our platform might have hiccups and respond slower than that).